lib/ansible/utils/module_docs_fragments/utm.py

```diff
@@ -0,0 +1,40 @@
+# Copyright: (c) 2018, Johannes Brunswicker <johannes.brunswicker@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+    DOCUMENTATION = """
+options:
+    utm_host:
+        description:
+          - The REST Endpoint of the Sophos UTM
```

Comment:
Period at end of each description statement.

Reply:
fixed